gst/gst_buffer.go

```diff
@@ -367,66 +367,14 @@ func (b *Buffer) Extract(offset, size int64) []byte {
 // FillBytes adds the given byte slice to the buffer at the given offset. The return value reflects the amount
 // of data added to the buffer.
 func (b *Buffer) FillBytes(offset int64, data []byte) int64 {
-	str := string(data)
-	cStr := C.CString(str)
-	gsize := C.gst_buffer_fill(b.Instance(), C.gsize(offset), (C.gconstpointer)(unsafe.Pointer(cStr)), C.gsize(len(str)))
-	return int64(gsize)
-}
-
-// FillInt8Slice adds the given slice to the buffer at the given offset. The return value reflects the amount
-// of data added to the buffer.
-func (b *Buffer) FillInt8Slice(offset int64, data []int8) int64 {
```

Comment:
Getting rid of these as I am not sure if they were implemented correctly and I feel like most users would be working with `[]byte` sequences anyway. May consider reimplementing in the future.

Reply:
Indeed I think keep just the FillBytes is good enough.